Resources/lang/fr/common.php

```diff
@@ -59,56 +59,56 @@
     // Aircraft - Pirep - Flight
     'altitude'  => 'Altitude',
     'avg'       => 'Moyenne',
-    'btime'     => 'Temps de vol (block)',
+    'btime'     => 'Durée de vol (block)',
     'config'    => 'Configuration',
     'dest'      => 'Destination',
     'dest_abr'  => 'Dest',
     'dist'      => 'Distance',
     'flightno'  => 'N° de Vol #',
     'flightid'  => 'Identifiant du vol (Code & Etape)',
-    'ftime'     => 'temps de vol',
-    'fuel'      => 'Carburant',
-    'fuelob'    => 'Carburant à bord',
-    'fuelused'  => 'Carburant utilisé',
+    'ftime'     => 'Heures de vol',
+    'fuel'      => 'Fuel',
+    'fuelob'    => 'Fuel à bord',
+    'fuelused'  => 'Fuel utilisé',
     'lastlnd'   => 'Dernier atterrissage',
-    'lrate'     => 'Taux de touché',
+    'lrate'     => 'Taux de descente',
     'orig'      => 'Origine',
     'orig_abr'  => 'Orig',
     'pic'       => 'Pilote aux Commandes',
     'reg'       => 'Immatriculation',
     'score'     => 'Score',
     'speed'     => 'Vitesse',
     'sta'       => 'Heure d\'arr.',
-    'std'       => 'Heure de dep.',
+    'std'       => 'Heure de dép.',
     'submitted' => 'Envoyé',
     // Airline - Airport - Hub
-    'aldetails' => 'Info sur le compagnie',
+    'aldetails' => 'Info sur la compagnie',
     'apdetails' => 'Détails de l\'aéroport',
     'downloads' => 'Téléchargements',
-    'fuelc'     => 'Prix du carburant',
-    'groundhc'  => 'Prix de la manutention au sol',
+    'fuelc'     => 'Prix carburant',
+    'groundhc'  => 'Prix des services au sol',
     'haircraft' => 'Hub Avions',
     'harrs'     => 'Hub Arrivées',
     'hdeps'     => 'Hub Départs',
     'hdetails'  => 'Hub Détails',
     'hpilots'   => 'Hub Pilotes',
-    'vaircraft' => 'Avions en Vérification',
-    'vpilots'   => 'Pilote en Vérification',
+    'vaircraft' => 'Avion en inspection',
+    'vpilots'   => 'Pilote en Instruction',
     // Finance
     'balance'   => 'Solde',
     'credit'    => 'Crédit',
     'debit'     => 'Débit',
-    'expense'   => 'Frais',
-    'finance'   => 'Financement Global',
-    'income'    => 'Revenu',
+    'expense'   => 'Dépenses',
+    'finance'   => 'Bilan général',
+    'income'    => 'Recettes',
     'journal'   => 'Opérations de journal',
     'memo'      => 'Détails',
     // Ranks
-    'allowed_sf'  => 'Sous-flotte Autoriée:',
-    'allowed_sh'  => 'Afficher/Masquer les Sous-flotte Autorisés',
+    'allowed_sf'  => 'Sous-flotte admise:',
+    'allowed_sh'  => 'Afficher/Masquer les Sous-flotte admises',
     'minhour'     => 'Heures Minimum',
-    'pay_manual'  => 'Salaire/Heure Manual',
-    'pay_acars'   => 'Salaire/Heure Acars',
+    'pay_manual'  => 'Salaire/H vol libre',
```

Comment:
I don't really know how to talk about the pay for manual pireps but "vol libre" is a "free flight" so i don't think it's perfect. Maybe "Salaire / H vol pour PIREPs Manuels" but it's not perfect too